JUnit version 3.8.1
.E.E..E.E..E..E.E.....E.
Time: 4.886
There were 8 failures:
1) test07(org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest)
java.lang.NullPointerException
	at org.apache.commons.jxpath.ri.axes.PredicateContext.reset(PredicateContext.java:168)
	at org.apache.commons.jxpath.ri.axes.PredicateContext.nextSet(PredicateContext.java:173)
	at org.apache.commons.jxpath.ri.EvalContext.performIteratorStep(EvalContext.java:154)
	at org.apache.commons.jxpath.ri.EvalContext.hasNext(EvalContext.java:115)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.containsMatch(CoreOperationRelationalExpression.java:123)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.compute(CoreOperationRelationalExpression.java:85)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.computeValue(CoreOperationRelationalExpression.java:46)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest.test07(CoreOperationRelationalExpression_ESTest.java:223)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
2) test06(org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest)
java.lang.ArithmeticException: / by zero
	at org.apache.commons.jxpath.ri.compiler.CoreOperationMod.computeValue(CoreOperationMod.java:42)
	at org.apache.commons.jxpath.ri.compiler.CoreOperation.compute(CoreOperation.java:56)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.computeValue(CoreOperationRelationalExpression.java:46)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest.test06(CoreOperationRelationalExpression_ESTest.java:197)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
3) test08(org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest)
java.lang.NullPointerException
	at org.apache.commons.jxpath.ri.axes.SelfContext.getSingleNodePointer(SelfContext.java:47)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.reduce(CoreOperationRelationalExpression.java:108)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.compute(CoreOperationRelationalExpression.java:73)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.computeValue(CoreOperationRelationalExpression.java:46)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest.test08(CoreOperationRelationalExpression_ESTest.java:251)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
4) test03(org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest)
java.lang.UnsupportedOperationException
	at org.apache.commons.jxpath.ri.axes.RootContext.nextSet(RootContext.java:89)
	at org.apache.commons.jxpath.ri.EvalContext.nextSet(EvalContext.java:339)
	at org.apache.commons.jxpath.ri.EvalContext.getSingleNodePointer(EvalContext.java:311)
	at org.apache.commons.jxpath.ri.InfoSetUtil.doubleValue(InfoSetUtil.java:125)
	at org.apache.commons.jxpath.ri.InfoSetUtil.doubleValue(InfoSetUtil.java:121)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.compute(CoreOperationRelationalExpression.java:90)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.containsMatch(CoreOperationRelationalExpression.java:125)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.findMatch(CoreOperationRelationalExpression.java:151)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.compute(CoreOperationRelationalExpression.java:82)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.computeValue(CoreOperationRelationalExpression.java:46)
	at org.apache.commons.jxpath.ri.compiler.CoreOperation.compute(CoreOperation.java:56)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.computeValue(CoreOperationRelationalExpression.java:46)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest.test03(CoreOperationRelationalExpression_ESTest.java:132)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
5) test02(org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest)
org.apache.commons.jxpath.JXPathInvalidSyntaxException: Incorrect number of arguments: count()
	at org.apache.commons.jxpath.ri.compiler.CoreFunction.assertArgRange(CoreFunction.java:926)
	at org.apache.commons.jxpath.ri.compiler.CoreFunction.assertArgCount(CoreFunction.java:915)
	at org.apache.commons.jxpath.ri.compiler.CoreFunction.functionCount(CoreFunction.java:353)
	at org.apache.commons.jxpath.ri.compiler.CoreFunction.computeValue(CoreFunction.java:252)
	at org.apache.commons.jxpath.ri.compiler.CoreFunction.compute(CoreFunction.java:242)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.computeValue(CoreOperationRelationalExpression.java:46)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest.test02(CoreOperationRelationalExpression_ESTest.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
6) test05(org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest)
java.lang.ArrayIndexOutOfBoundsException: 0
	at org.apache.commons.jxpath.ri.compiler.CoreOperationSubtract.computeValue(CoreOperationSubtract.java:40)
	at org.apache.commons.jxpath.ri.compiler.CoreOperation.compute(CoreOperation.java:56)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.computeValue(CoreOperationRelationalExpression.java:46)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest.test05(CoreOperationRelationalExpression_ESTest.java:178)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
7) test04(org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest)
org.evosuite.runtime.mock.java.lang.MockIllegalArgumentException: No such variable: ''
	at org.apache.commons.jxpath.BasicVariables.getVariable(BasicVariables.java:62)
	at org.apache.commons.jxpath.ri.model.VariablePointer.getBaseValue(VariablePointer.java:79)
	at org.apache.commons.jxpath.ri.model.VariablePointer.getLength(VariablePointer.java:143)
	at org.apache.commons.jxpath.ri.axes.InitialContext.setPosition(InitialContext.java:69)
	at org.apache.commons.jxpath.ri.axes.InitialContext.nextNode(InitialContext.java:63)
	at org.apache.commons.jxpath.ri.EvalContext.performIteratorStep(EvalContext.java:155)
	at org.apache.commons.jxpath.ri.EvalContext.hasNext(EvalContext.java:115)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.findMatch(CoreOperationRelationalExpression.java:147)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.compute(CoreOperationRelationalExpression.java:82)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.computeValue(CoreOperationRelationalExpression.java:46)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest.test04(CoreOperationRelationalExpression_ESTest.java:156)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
8) test00(org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest)
org.apache.commons.jxpath.JXPathException: Cannot invoke extension function org.apache.commons.jxpath.ri.model.container.ContainerPointerFactory; Class 'org/apache/commons/jxpath/ri/model/container.class' should be in target project, but could not be found!
	at org.apache.commons.jxpath.PackageFunctions.getFunction(PackageFunctions.java:192)
	at org.apache.commons.jxpath.ri.JXPathContextReferenceImpl.getFunction(JXPathContextReferenceImpl.java:755)
	at org.apache.commons.jxpath.ri.axes.RootContext.getFunction(RootContext.java:140)
	at org.apache.commons.jxpath.ri.compiler.ExtensionFunction.computeValue(ExtensionFunction.java:96)
	at org.apache.commons.jxpath.ri.compiler.ExtensionFunction.compute(ExtensionFunction.java:84)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression.computeValue(CoreOperationRelationalExpression.java:46)
	at org.apache.commons.jxpath.ri.compiler.CoreOperationRelationalExpression_ESTest.test00(CoreOperationRelationalExpression_ESTest.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: Class 'org/apache/commons/jxpath/ri/model/container.class' should be in target project, but could not be found!
	at org.evosuite.runtime.instrumentation.EvoClassLoader.instrumentClass(EvoClassLoader.java:123)
	at org.evosuite.runtime.instrumentation.EvoClassLoader.loadClass(EvoClassLoader.java:92)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at org.apache.commons.jxpath.util.ClassLoaderUtil.getClass(ClassLoaderUtil.java:94)
	at org.apache.commons.jxpath.util.ClassLoaderUtil.getClass(ClassLoaderUtil.java:149)
	at org.apache.commons.jxpath.PackageFunctions.getFunction(PackageFunctions.java:189)
	... 18 more
Caused by: java.lang.ClassNotFoundException: Class 'org/apache/commons/jxpath/ri/model/container.class' should be in target project, but could not be found!
	at org.evosuite.runtime.instrumentation.EvoClassLoader.instrumentClass(EvoClassLoader.java:109)
	... 24 more

FAILURES!!!
Tests run: 16,  Failures: 8

